docs_source_files/content/worst_practices/two_factor_authentication.zh-cn.md

```diff
@@ -9,18 +9,13 @@ English to Chinese. Do you speak Chinese? Help us to translate
 it by sending us pull requests!
 {{% /notice %}}
```

Comment:
I think this header should be removed if the translation is done!

Reply:
i think so